src/checkers/inference/DefaultSlotManager.java

```diff
@@ -156,14 +156,10 @@ public AnnotationMirror getAnnotation( final Slot slot ) {
         // We need to build the AnntotationBuilder each time because AnnotationBuilders are only allowed to build their annotations once
         if (slotClass.equals(VariableSlot.class) || slotClass.equals(ExistentialVariableSlot.class)
                 || slotClass.equals(RefinementVariableSlot.class) || slotClass.equals(CombVariableSlot.class)
-                || slotClass.equals(ConstantSlot.class)) {
+                || slotClass.equals(ConstantSlot.class) || slotClass.equals(ConstantSlot.class)) {
```

Comment:
The current implementation of `getAnnotation(final Slot slot)` DOES already return the `VarAnnot` for ConstantSlot. For example:
If we call this method on a ConstantSlot - `GUT.qual.Rep`, it returns `@checkers.inference.qual.VarAnnot(5)`.
The existing implementation already has `slotClass.equals(ConstantSlot.class)` in the if statement, which correctly returns the `VarAnnot` for ConstantSlot. You can see there are two 'slotClass.equals(ConstantSlot.class)' conditions in your fix, which is duplicate.

Reply:
Ah, I just saw the if-statement specially for `ConstantSlot` below, and didn't have a check on what existed on this if-statement. Ok, I will remove the duplicate check.

Thanks for explanation!